packages/ui/src/working-groups/hooks/useGroupDebt.ts

```diff
@@ -0,0 +1,21 @@
+import BN from 'bn.js'
+
+import { WorkerStatusTypename } from '@/working-groups/types'
+
+import { useGetGroupDebtQuery } from '../queries'
+
+export const useGroupDebt = (groupId: string) => {
+  const { data, loading } = useGetGroupDebtQuery({
+    variables: { where: { group_eq: groupId, status_json: { isTypeOf_in: [WorkerStatusTypename['active']] } } },
+  })
+
+  if (loading || !data) {
+    return {
+      debt: null,
+    }
+  }
+
+  return {
+    debt: new BN(data.workers.reduce((a, b) => a + b.missingRewardAmount, 0)),
```

Comment:
Depending on how much debt can be racked up, `BN` might be a better option for an accumulator than `number` - not too important yet but might be worth keeping in mind

Reply:
It might be a good option. A bit future proof as well :+1: